app/views/searches/_results.html.haml

```diff
@@ -17,4 +17,4 @@
           %td.govuk-table__cell= result.prosecution_case_reference
           %td.govuk-table__cell= result.first_name
           %td.govuk-table__cell= result.last_name
-          %td.govuk-table__cell= l(result.date_of_birth.to_date)
+          %td.govuk-table__cell= l(result.date_of_birth.to_date) if result.date_of_birth
```

Comment:
This looks good, however, based on my knowledge of the data structure and search, shouldn't a DoB always be available?

Reply:
should be, but code defensively. came up through use of local version of mock with factory generated values. i18n throws errors on the localize method with nil args